core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form class="form-inline" ng-submit="addExploration()">
-  <input class="form-control protractor-test-add-exploration-input" ng-model="newExplorationId" type="text" placeholder="Existing exploration ID">
-  <button class="btn btn-default protractor-test-add-exploration-button" type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-control protractor-test-add-exploration-input" ng-model="newExplorationId" type="text" placeholder="Search explorations by name" typeahead="exploration for exploration in fetchExplorationsMetadata($viewValue)">
```

Comment:
For this situation, we have two options.
- User's can search using the exploration id (which doesn't seems appropriate to me because if I'm not a developer -- I'll feel a bit uncomfortable with handling exploration ids).
- User's cannot search using exploration id and also, they can't directly type their exploration id and press enter to add it to the collection. They'll have to select an exploration from the search result.

I think, the second option is better. What do you think about it ?